### PR TITLE
Enhancement: Use system timezone if no TIMEZONE env

### PR DESCRIPTION
**Pull request by dapi**
Have a nice day! )

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    config.time_zone = ENV.fetch('TIMEZONE')
+    config.time_zone = ENV.fetch('TIMEZONE', File.read('/etc/timezone').strip)
```
Comment (mnaichuk) [inline config/application.rb:40]: we have this one on master `config.time_zone = ENV.fetch('TIMEZONE', 'UTC')` so by default we will use UTC time.

Comment (dapi) [reply config/application.rb:40]: Why? )

Comment (calj) [reply config/application.rb:40]: Myself I would even hardcode 'UTC' and not let people change it.
It's a very bad idea to change the timezone from backend.
The good way to work with timezone (and the only one to me) is to configure backend software and database in UTC and then change the time zone based on end user preference on display.

Comment (dapi) [reply config/application.rb:40]: @calj thank you for explanation!

One point to merge this request. Peatio raises an error if `TIMEZONE` is absent.
Just want to decrease time and number of movements to get peatio server works from scratch.

Comment (calj) [reply config/application.rb:40]: yes agree we should backport this patch

**Pull request closed without merging**